ST/30_api_model_rest_basics/Api_Model_and_Rest_Basics_README.md

```diff
@@ -0,0 +1,391 @@
+
+<p style="font-size: 45px; text-align: center;">
+<strong> API Model. Rest Basics </strong>
+</p>
+
+<div style="text-align: center;">
+    <img width="25%" src="images/api-anim.gif">
+</div>
+
+
+***
+
+## Contents
+
+1. [Brief Overview of API Testing and Importance of Automated API Testing](#introduction-to-api-and-importance)
```

Comment:
Please check if links of your Table of Contents are working properly

Reply:
Done
Thanks